scripts/missions/cop/7_5_The_Warriors_Path.lua

```diff
@@ -0,0 +1,150 @@
+require('scripts/globals/npc_util')
+require('scripts/globals/titles')
+require('scripts/globals/utils')
+require('scripts/globals/zone')
+-----------------------------------
+local altaieuID = require("scripts/zones/AlTaieu/IDs")
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.COP, xi.mission.id.cop.THE_WARRIORS_PATH)
+
+mission.reward =
+{
+    title = xi.title.SEEKER_OF_THE_LIGHT,
+    nextMission = { xi.mission.log_id.COP, xi.mission.id.cop.GARDEN_OF_ANTIQUITY },
+}
+
+mission.sections =
+{
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId
+        end,
+
+        [xi.zone.SEALIONS_DEN] =
+        {
+            ['_0w0'] =
+            {
+                onTrigger = function(player, npc)
+                    if mission:getVar(player, 'Status') == 0 then
+                        return mission:progressEvent(32)
+                    end
+                end,
+            },
+
+            ['Sueleen'] = mission:event(28),
+
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    if mission:getVar(player, 'Status') == 2 then
+                        return 34
+                    end
+                end,
+            },
+
+            onEventUpdate =
+            {
+                [32] = function(player, csid, option, npc)
+                    if option == 0 then
+                        player:updateEvent(32, 0, 0, 0, 0, 0, 0, 0)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [32] = function(player, csid, option, npc)
+                    mission:setVar(player, 'Status', 1)
+                end,
+
+                [34] = function(player, csid, option, npc)
+                    mission:setVar(player, 'Status', 3)
+                    player:setPos(-422.100, 0, -532.092, 220, xi.zone.ALTAIEU)
+                end,
+
+                [32001] = function(player, csid, option, npc)
+                    if
+                        mission:getVar(player, 'Status') == 1 and
+                        player:getLocalVar('battlefieldWin') == 993
+                    then
+                        mission:setVar(player, 'Status', 2)
+                        player:setPos(612.057, 132.664, 776.920, 188, xi.zone.SEALIONS_DEN)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.ALTAIEU] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    if mission:getVar(player, 'Status') == 3 then
+                        return 1 -- Event Options 1 and 2 come for update requests, no observed responses
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [1] = function(player, csid, option, npc)
+                    -- Nag'molada steals one random light
```

Comment:
Note to self: This isn't a random light, but aligns with player race.
![image](https://user-images.githubusercontent.com/81713309/167017989-e941a2b8-f005-4e62-af56-1a3903eb794d.png)

Galka:
```
[16:36:47] You obtain the light of Al'Taieu, only to have it stolen by Nag'molada...
```

Reply:
The Galka message is also a totally different one than the non-galka, because you never "obtain" any light. 

Galka:
Al'Taieu given & stolen, one line

Hume:
Vazhl taken (line 1)
Al'Taieu given (line 2)